src/V3EmitV.cpp

```diff
@@ -598,12 +598,18 @@ class EmitVBaseVisitor VL_NOT_FINAL : public EmitCBaseVisitor {
         iterate(nodep->subDTypep());
         iterateAndNextConstNull(nodep->rangep());
     }
+    void visit(AstRefDType* nodep) override {
+	iterate(nodep->skipRefp());
+    }
     void visit(AstNodeUOrStructDType* nodep) override {
         puts(nodep->verilogKwd() + " ");
         if (nodep->packed()) puts("packed ");
         puts("\n");
         puts("{");
-        iterateAndNextConstNull(nodep->membersp());
+        for (AstMemberDType* itemp = nodep->membersp(); itemp; itemp = VN_AS(itemp->nextp(), MemberDType)) {
+            iterate(itemp);
+            puts(";");
```

Comment:
@wsnyder I simplified this PR greatly. An anonymous struct was already being generated as part of the port. It just needed a `;` between members to be correctly formed.

Reply:
That certainly simplified it, much more than I had considered, thanks.